ingress/validation.go

```diff
@@ -56,6 +56,9 @@ func ValidateForSerialization(info *livekit.IngressInfo) error {
 		if info.Url == "" {
 			return ErrInvalidIngress("no source URL")
 		}
+		if info.Enabled != nil && !*info.Enabled {
+			return ErrIngressDisabled
```

Comment:
Please return `ErrInvalidIngress("disabled non reusable ingress")`, or a similar message. The issue here is that the ingress has an inconsistent set of parameters, not that is it merely disabled.